etcdctlv3/command/txn_command.go

```diff
@@ -41,198 +51,178 @@ func txnCommandFunc(cmd *cobra.Command, args []string) {
 		ExitWithError(ExitBadArgs, fmt.Errorf("txn command does not accept argument."))
 	}
 
+	if !txnInteractive {
+		ExitWithError(ExitBadFeature, fmt.Errorf("txn command only supports interactive mode"))
+	}
+
 	reader := bufio.NewReader(os.Stdin)
 
-	next := compareState
-	txn := &pb.TxnRequest{}
-	for next != nil {
-		next = next(txn, reader)
-	}
+	txn := clientv3.NewKV(mustClientFromCmd(cmd)).Txn(context.Background())
+	fmt.Println("entry comparison[key target expected_result compare_value] (end with empty line):")
+	txn.If(readCompares(reader)...)
+	fmt.Println("entry success request[method key value(end_range)] (end with empty line):")
+	txn.Then(readOps(reader)...)
+	fmt.Println("entry failure request[method key value(end_range)] (end with empty line):")
```

Comment:
using `success requests (get, put, delete):`; wordy hint would be cluttered and probably wouldn't be any more helpful